src/gui/folderwizard/folderwizardselectivesync.cpp

```diff
@@ -43,16 +43,14 @@ FolderWizardSelectiveSync::FolderWizardSelectiveSync(FolderWizardPrivate *parent
     _selectiveSync = new SelectiveSyncWidget(folderWizardPrivate()->accountState()->account(), this);
     layout->addWidget(_selectiveSync);
 
-    const auto vfsMode = VfsPluginManager::instance().bestAvailableVfsMode();
-    if (Theme::instance()->forceVirtualFilesOption() && vfsMode == Vfs::WindowsCfApi) {
-        // using an else if to make the condition not even more complex...
-    } else if (Theme::instance()->showVirtualFilesOption() && vfsMode != Vfs::Off && (vfsMode == Vfs::WindowsCfApi || Theme::instance()->enableExperimentalFeatures())) {
-        _virtualFilesCheckBox = new QCheckBox(tr("Use virtual files instead of downloading content immediately%1").arg(vfsMode == Vfs::WindowsCfApi ? QString() : tr(" (experimental)")));
+    if (!Theme::instance()->forceVirtualFilesOption() && Theme::instance()->showVirtualFilesOption()
+        && VfsPluginManager::instance().bestAvailableVfsMode() == Vfs::WindowsCfApi) {
```

Comment:
In the long term, we probably shouldn't hardcode the Windows implementation here, but just have `bestAvailableVfsMode` return `Vfs::Off` even if we ship such plugins.

I guess this needs some more refactoring in 7.x.